boxes/templates/boxes/box_list.html

```diff
@@ -164,64 +156,43 @@ <h3 class="subtitle center-xs">Create New Box</h3>
 
             </div>
           </div>
-
           <div class="row bottom-xs">
             <div class="col-xs-9 col-xxs-12 col-sm-8">
               <p>
                 {{ form.expires_at.errors }}
                 {{ form.expires_at.label_tag }}
                 {{ form.expires_at }}
-              </p>   
+              </p>
             </div>
             <div class="col-xs-3 col-xxs-12 col-sm-4  ">
              <div class="row">
               <div class="col-xs-10 col-xxs-8 sm-10">
                 <p> {{ form.never_expires.errors }}
                 </div>
-                {{ form.never_expires.label_tag }}</p>   
+                {{ form.never_expires.label_tag }}</p>
                 <div class="col-xs-2 col-xxs-4 sm-2">
                   {{ form.never_expires }}</div>
 
                 </div> </div>
               </div>
             </div>
-
-          </label>         
+          </label>
         </div>
-
         <div class="row middle-xs center-xs">
           <div class="col-xs-12">
             <button type="submit" class="btn-default" value=""> Create</button>
-
           </div>
-
         </div>
       </form>
-
     </div>
-
-
   </div>
   <div class="md-overlay"></div>
 </div>
-
-
 {% endblock content %}
 
 {% block scripts %}
-
-<script src="{% static 'javascripts/box_list.js' %}"></script>
-
-
-<script type="text/javascript" src="{% static 'javascripts/jquery.datetimepicker.min.js' %}"></script>
-<script src="{% static 'javascripts/box_create.js' %}"></script>
-
-
-
+  {{ block.super }}
```

Comment:
@Budgerous Can you please clarify what you meant with your comment? Does it affect this line in particular?